File: Modules/SMES/MES/SIE.Web.MES/PrepareProducts/Commands/PrepareProductDetailAddCommand.js
SIE.defineCommand("SIE.Web.MES.PrepareProducts.Commands.PrepareProductDetailAddCommand", {
    extend: "SIE.cmd.Add",
    meta: { text: "添加", group: "edit", iconCls: "icon-AddEntity icon-green" },
    onItemCreated: function (entity) {
        var me = this;
        var parent = me.view.getParent().getCurrent();
        entity.setProFamiliyId(parent.getProductFamilyId());
        entity.setPrepareProductId(parent.getId());
    },
});

File: Modules/SMES/MES/SIE.Web.MES/PrepareProducts/Commands/PrepareProductDetailAddCommand.cs
using SIE.Web.Command;
using System;
using System.Collections.Generic;
using System.Text;

namespace SIE.Web.MES.PrepareProducts.Commands
{
    /// <summary>
    /// 产品产前准备子表添加命令
    /// </summary>
    public class PrepareProductDetailAddCommand : ViewCommand
    {
        /// <summary>
        /// 执行
        /// </summary>
        /// <param name="args"></param>
        /// <param name="scope"></param>
        /// <returns></returns>
        /// <exception cref="NotImplementedException"></exception>
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}
